src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -24,16 +24,17 @@ private ChatPageObject openChatForUser(WebDriver driver, String userName, String
   @DontRun(env = {"sandbox"})
   @Test(groups = {"ChatTestsStaff_001", "ChatStaff"})
   public void ChatTestsStaff_001_twoUserEnterChat() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)